mmv1/templates/terraform/post_create/certificatemanager_certificate.go.erb

```diff
@@ -0,0 +1,16 @@
+<%- # the license inside this block applies to this file
+	# Copyright 2023 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+	// Force legacy id format for global triggers.
+	id = strings.ReplaceAll(id, "/locations/global/", "/")
```

Comment:
note: I think the old `id` string was `projects/{{project}}/locations/global/certificates/{{name}}` so this replacement probably isn't necessary!

Reply:
Ah- preserving the old import format may be the issue. We may want to define a `custom_import` to compartmentalize the logic if so (since I think we could eliminate the `pre_read` too)